code/datums/flock/flockunlockable.dm

```diff
@@ -52,11 +53,11 @@ ABSTRACT_TYPE(/datum/unlockable_flock_structure)
 	structType = /obj/flock_structure/compute
 
 	check_unlocked()
-		return src.my_flock.hasAchieved("all_structures")
+		return src.my_flock.hasAchieved(FLOCK_ACHIEVEMENT_CHEAT_STRUCTURES)
 
 /datum/unlockable_flock_structure/gnesisturret
 	structType = /obj/flock_structure/gnesisturret
 
 	check_unlocked()
-		return src.my_flock.hasAchieved("human dissection") || src.my_flock.hasAchieved("all_structures")
+		return src.my_flock.hasAchieved(FLOCK_ACHIEVEMENT_CAGE_HUMAN) || src.my_flock.hasAchieved(FLOCK_ACHIEVEMENT_CHEAT_STRUCTURES)
```

Comment:
```dm
return ..() || src.my_flock.hasAchieved(FLOCK_ACHIEVEMENT_CAGE_HUMAN)
```
can't something like this be done and then move `src.my_flock.hasAchieved(FLOCK_ACHIEVEMENT_CHEAT_STRUCTUERS)` to the parent?

Reply:
I guess - is there any point though? seems like it'd just add more proc overhead and not really fix the issue of needing to copy & paste when you want to define a new structure unlock

Reply:
It feels a little cleaner and I have feeling this is preferred by imcoder to copy+pasting conditions everywhere. Probably best to ask there. If they say otherwise my review's moot and it fine to be wordy.